packages/server/jest.config.ts

```diff
@@ -16,15 +16,10 @@ const baseConfig: Config.InitialProjectOptions = {
     "@budibase/backend-core": "<rootDir>/../backend-core/src",
     "@budibase/shared-core": "<rootDir>/../shared-core/src",
     "@budibase/types": "<rootDir>/../types/src",
+    "@budibase/pro": "<rootDir>/../pro/packages/pro/src",
```

Comment:
This may break open source contributors who don't have access to the pro repo. Without the source mapping they will get the types only build of pro from npm + the minified bundle. If we include this they'll likely be in a broken state

Reply:
Yup, when everything runs for us I will be checking what is the 'no-pro' experience 👍🏽

Reply:
Done